src/test/fuzz/parse_univalue.cpp

```diff
@@ -77,7 +77,7 @@ FUZZ_TARGET(parse_univalue, .init = initialize_parse_univalue)
     }
     try {
         FlatSigningProvider provider;
-        (void)EvalDescriptorStringOrObject(univalue, provider);
+        if (buffer.size() < 10'000) (void)EvalDescriptorStringOrObject(univalue, provider);
```

Comment:
would it make sense to use a trimmed version here instead of completely skipping the call?